lib/l10n/arb/app_pt.arb

```diff
@@ -0,0 +1,225 @@
+{
+  "@@locale": "pt",
+  "play": "Jogar",
+  "@play": {
+    "description": "Texto exibido no botão de jogar da página inicial"
+  },
+  "howToPlay": "Como Jogar",
+  "@howToPlay": {
+    "description": "Texto exibido no botão de como jogar da página inicial"
+  },
+  "tipsForFlips": "Dicas para flips",
```

Comment:
```suggestion
  "tipsForFlips": "Dicas para arremesso",
```

Reply:
I was searching for how flips are called in Portuguese! Thanks Pedro